client/data/emails/use-emails-query.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import { useQuery } from 'react-query';
+
+/**
+ * Internal dependencies
+ */
+import wpcom from 'calypso/lib/wp';
+
+const getCacheKey = ( siteId, domain ) => [ 'emails', siteId, domain ];
```

Comment:
It might be worth it to add `accounts` in there to better match the URL below.

Reply:
Sounds like a good idea.